src/cli/utils/utils.go

```diff
@@ -38,14 +39,19 @@ func GetAccountAndConnect(
 		return nil, nil, nil, clues.Wrap(err, "connecting to the "+cfg.Storage.Provider.String()+" repository")
 	}
 
+	if _, err := r.ConnectToM365(ctx, pst); err != nil {
```

Comment:
What's the purpose of this call? Is this like a dry run/sanity check on m365 connection?

Reply:
It will initialize the graph api clients, including enabling the concurrency limiter.  I'll add a comment for clarification.